src/pages/Competition/Information.tsx

```diff
@@ -76,17 +76,19 @@ const UserRow = ({ user }: { user: User }) => {
       <a
```

Comment:
What happens if someone clicks on the link? We should likely change the type of this root component to a div if there's nothing to link to.

Reply:
Actually seems pretty good.